platforms/Linux/DEB/Ubuntu/focal/debian/control.in

```diff
@@ -0,0 +1,46 @@
+Source: swiftlang
+Section: devel
+Priority: optional
+Maintainer: Swift Maintainers <swift-infrastructure@forums.swift.org>
+Build-Depends: debhelper-compat (= 12), devscripts, 
+ clang, cmake, git, icu-devtools, ninja-build, pkg-config,
+ libcurl4-openssl-dev, libedit-dev, libicu-dev, libncurses5-dev,
+ libpython3-dev, libsqlite3-dev, libxml2-dev,
+ python, python-six, python2-dev, python3-six, python3-distutils, python3-psutil,
+ rsync, swig, systemtap-sdt-dev, tzdata, unzip, uuid-dev
+Standards-Version: 4.4.1
+Homepage: https://www.swift.org
+
+Package: swiftlang
+Architecture: all
+Depends: swiftlang-x.y.z, ${misc:Depends}
+Description: Swift programming language
+ Swift is a general-purpose programming language built using
+ a modern approach to safety, performance, and software design
+ patterns.
+ .
+ The goal of the Swift project is to create the best available
+ language for uses ranging from systems programming, to mobile
+ and desktop apps, scaling up to cloud services. Most
+ importantly, Swift is designed to make writing and maintaining
+ correct programs easier for the developer.
+ .
+ This package contains the convenience symlinks for the latest
+ version of Swift.
+
+Package: swiftlang-x.y.z
+Architecture: amd64 arm64
+Depends: ${shlibs:Depends}, ${misc:Depends},
```

Comment:
sort dependencies alphabetically?

Reply:
I grouped them by category (and not necessarily alphabetically within those) out of habit. It's usually easier to maintain this way.